### PR TITLE
[#308] add tests and message for deleting workspaces

### PR DESCRIPTION
**Pull request by Tereza-Pytelova**

### CODE REVIEW

File: scripts/workspace_delete.py [hunk 1]
```diff
@@ -70,12 +76,17 @@ def main(argv):
             logger.error("Error while deleting workspace  '%s', status code '%s'", workspace['name'], response.status_code)
             sys.exit(1)
 
-    logger.info("All workspaces were successfully deleted")
+    if not nWorkspacesDeleted:
+        logger.info("No workspaces has been deleted")
+    elif nWorkspacesDeleted == 1:
```
[the remark this answers is not in the document]
Comment (ondrejtomas) [reply scripts/workspace_delete.py]: I found that we log this info on line 68, so we don't have to create an issue for that.

[the remark this answers is not in the document]
Comment (Tereza-Pytelova) [reply scripts/workspace_delete.py]: Great ;)

File: ci/unit_tests/workspace_delete/main_test.py [hunk 2]
```diff
@@ -230,13 +230,55 @@ def test_deleteAllWithRegex(self, envVarNameUsername, envVarNamePassword):
...
+        self.t_noExceptionAndLogMessage("3 workspaces have been successfully deleted",[deleteParams])
 
         workspaces = getWorkspaces(self.workspacesUrl, self.version, self.username, self.password)
 
         # there should be no workspace left
         assert len(workspaces) == 0
 
+    @pytest.mark.parametrize('envVarNameUsername, envVarNamePassword', [('WA_USERNAME', 'WA_PASSWORD')])
+    def test_deleteByNameNoneMatching(self, envVarNameUsername, envVarNamePassword):
+        """Tests if no workspace match name."""
```
[the remark this answers is not in the document]
Comment (Tereza-Pytelova) [reply ci/unit_tests/workspace_delete/main_test.py]: fixed

File: ci/unit_tests/workspace_delete/main_test.py [hunk 3]
```diff
@@ -230,13 +230,55 @@ def test_deleteAllWithRegex(self, envVarNameUsername, envVarNamePassword):
...
+        workspaceName = "My skill"
+        deployParams = list(self.deployParamsBase)
+        deployParams.extend(['--conversation_workspace_name', workspaceName])
+        workspace_deploy.main(deployParams)
+
+        # try to delete non-matching workspace
+        deleteParams = list(self.deleteParamsBase)
+        deleteParams.extend(['--conversation_workspace_match_by_name', 'true',
+                             '--conversation_workspace_name_pattern', 'workspace'])
+        self.t_noExceptionAndLogMessage("No workspaces has been deleted",[deleteParams])
```
[the remark this answers is not in the document]
Comment (Tereza-Pytelova) [reply ci/unit_tests/workspace_delete/main_test.py]: fixed

**Pull request merged**